Library/Formula/protobuf.rb

```diff
@@ -89,16 +102,31 @@ def install
     end
   end
 
+
   test do
-    def testdata; <<-EOS.undent
-      package test;
-      message TestCase {
-        required string name = 4;
-      }
-      message Test {
-        repeated TestCase case = 1;
-      }
-      EOS
+    if devel?
+      def testdata; <<-EOS.undent
```

Comment:
Use a variable instead of a function here:
```ruby
testdata = if devel?
  <<-EOS.undent
    ...
  EOS
else
  <<-EOS.undent
    ...
  EOS
end
```